README.md

```diff
@@ -31,66 +31,89 @@ The Perl container is set up so that Apache will load additionally provided .con
-dancer-app-1-build                                                                                                                                                     ip-10-230-142-143.ec2.internal/10.230.142.143   build=dancer-app-1,buildconfig=dancer-app,name=dancer-app,template=dancer-example   Succeeded    15 minutes   
-                                   sti-build      openshift/origin-sti-builder:v0.5.3                                                                                                                                                                                                                      Terminated   15 minutes   exit code 0
-frontend-1-w6cef     172.17.0.50                                                                                                                                       ip-10-230-142-143.ec2.internal/10.230.142.143   deployment=frontend-1,deploymentconfig=frontend,name=frontend                       Running      3 minutes    
-                                   dancer-app     172.30.221.178:5000/demo/origin-dancer-app@sha256:dd8452a64d8cf9ba36a461603306ae440b289f819c77dae1989c8980f3243f8e                                                                                                                                       Running      3 minutes    
+
+		$ oc get pods -w
+
+
+	Sample output:  
+
+		NAME                       READY     REASON    RESTARTS   AGE
+		dancer-example-1-build     1/1       Running   0          4m
+		dancer-frontend-1-deploy   1/1       Running   0          4s
+		dancer-frontend-1-votfl    0/1       Pending   0          1s
+		NAME                     READY     REASON       RESTARTS   AGE
+		dancer-example-1-build   0/1       ExitCode:0   0          4m
+		dancer-frontend-1-votfl   0/1       Running   0         6s
+		dancer-frontend-1-deploy   0/1       ExitCode:0   0         14s
+		dancer-frontend-1-votfl   1/1       Running   0         12s    
 
 
 6. Check the IP and port the frontend service is running on:  
-`oc get services`  
-Sample output:  
->NAME       LABELS                    SELECTOR        IP(S)            PORT(S)
-frontend   template=dancer-example   name=frontend   172.30.223.197   8080/TCP
 
-In this case, the IP for frontend is 172.30.223.197 and it is on port 8080.  
+		$ oc get svc
+
+	Sample output:  
+
+		NAME              LABELS                          SELECTOR               IP(S)            PORT(S)
+		dancer-frontend   template=dancer-mysql-example   name=dancer-frontend   172.30.174.142   8080/TCP
+
+In this case, the IP for frontend is 172.30.174.142 and it is on port 8080.  
 *Note*: you can also get this information from the web console.
 
 ###Manual Installation: With MySQL
 1. Create an account at [https://www.openshift.com](https://www.openshift.com)  
-*NOTE*: OpenShift Online currently is using V2.
+  -*NOTE*: OpenShift Online currently is using V2.  
```

Comment:
again remove and just tell them "we assume you have access to an openshift deployment"